.remarkrc.mjs

```diff
@@ -12,6 +12,10 @@ export default {
     // @see: https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-list-item-spacing
     ['remark-lint-list-item-spacing', false],
 
+    // Allow duplicate headings as it is completely valid to have headings of the same name in a document.
```

Comment:
**question (if-minor):** I wonder why? Can you please describe this more, why the duplication is valid?

Reply:
There are generic headings which – in my opinion – can be repeated across sections. For example "API", "Options", "Theming". If you are describing a few subcomponents, that's the case. In `Header`, there were two "Navigation" headings.

Or a CHANGELOG. I found out the reason it's ignored by Remark is that it is generated, but if it wasn't… 🙂 

Also, I noticed how you bypassed the limitation in our `web` READMEs by adding "Component" to conflicting headings 🙂. But then comes a question: shall we apply this form to all component headings, or just the conflicting ones?

It turns out that IDEs and Git hosts handle the duplicate headings the same way, that is, in case of conflict when generating HTML, they add a number suffix to their ID (at least PhpStorm and GH do it, I can check more).

So as I see it, it's more a question of linking the headings than the duplication itself. The question is whether it's more comfortable and sustainable for us to link to headings like `#card-1` (and, for me as a human reading the document, "Card" is what I want to see), or to be more specific and future proof in cases like `#cardbody-api` instead of `api-4`.

Reply:
I am ok with this. Can you please shorten this description and add it as a comment to the setting? So we will be more aware of this setting :-)

Reply:
Ok, I have added the comment.